bitcoin-magazine/bitcoin-explained/op_return.md

```diff
@@ -0,0 +1,596 @@
+And I think that was 172 bytes so you could have said okay we're just gonna make OP_RETURN 172 bytes because in that case there is really no reason to use these multi-sig keys anymore because you might as well use OP_RETURN but there was a simultaneous I guess movement or desire to not make the blockchain unnecessarily big.
+
+Aaron Van Wirdum: 00:12:21
+
+And
+
+Sjors Provoost: 00:12:22
+
+I think it's useful to look at that in perspective.
+So right now there is a fee market.
+So that means that if you're putting a lot of data onto the blockchain, it's going to be very expensive for you to do that.
+Or at least there's a mechanism to make sure that, you know, useful stuff goes on the blockchain more easily than useless stuff, depending on if you define useful by how much you're willing to pay for it.
+But back then, blocks were super small.
+They were maybe 10 kilobytes or 100 kilobytes, I don't know where they were in 2014.
+And that meant that this OP_RETURN data could just make blocks really big for the time that happened.
+And we've done an episode in the past where we looked at, I think it's episode number 55, where we looked at old node software and see how that old node software would perform under the current block sizes.
+And it turns out that very old node software, especially from say 2012, would not have been able to keep up with actual one megabyte blocks,
+
+Aaron Van Wirdum: 00:13:14
+
+Right.
+
+Sjors Provoost: 00:13:14
+
+or at least not very easily.
+So back then it was actually necessary to keep blocks small, basically from a more altruistic point of view.
+And that's what a lot of the discussion was about, like what is good behavior on the blockchain.
+
+Aaron Van Wirdum: 00:13:27
+
+So why was, so the first limit that was set, well initially the limit was zero, initially OP_RETURN was effectively impossible, but then the limit was increased to 40 and why why was it increased to 40 okay
+
+Sjors Provoost: 00:13:43
+
+so the idea behind 40 is that it allows you to put a hash in there, a typical hash using normal cryptography like SHA-256 is 32 bytes.
+So this allows you to put a 32 byte hash on the blockchain and then a few extra bytes to do, I don't know, some metadata for that hash.
+Now there were some protocols out there that wanted 80 bytes.
+
+Aaron Van Wirdum: 00:14:06
+
+Yeah, Counterparty in specific.
+This is what sort of this blog post that is the reason we're making this podcast.
+That's the main example that's given.
+It's basically about the counterparty protocol.
+
+Sjors Provoost: 00:14:17
+
+Yeah, so I don't know for sure if the increase to 80 bytes was only for counterparty or whether there was another protocol that needed it.
+But basically there was some back and forth and I guess people decided to just allow 80 bytes.
+But really 40 should be enough for most things, but it requires a little bit more work for those protocols.
+So if you look at something like, as far as I
```

Comment:
Here's another instance of a non-essential line. Since the speaker doesn't complete their thought, this doesn't contribute value to the transcript and can be removed. Additionally, the wrongly attributed 'know...' in the next line should also be deleted for accuracy.